accounts/abi/bind/lib.go

```diff
@@ -0,0 +1,152 @@
+package bind
+
+import (
+	"github.com/ethereum/go-ethereum"
+	"github.com/ethereum/go-ethereum/accounts/abi"
+	"github.com/ethereum/go-ethereum/common"
+	"github.com/ethereum/go-ethereum/core/types"
+	"github.com/ethereum/go-ethereum/crypto"
+	"github.com/ethereum/go-ethereum/event"
+)
+
+func DeployContract2(opts *TransactOpts, bytecode []byte, input []byte, backend ContractBackend) (common.Address, *types.Transaction, error) {
+	c := NewBoundContract(common.Address{}, abi.ABI{}, backend, backend, backend)
+	tx, err := c.transact(opts, nil, append(bytecode, input...))
+	if err != nil {
+		return common.Address{}, nil, err
+	}
+	address := crypto.CreateAddress(opts.From, tx.Nonce())
+	return address, tx, nil
+}
+
+func Call2[T any](opts *CallOpts, addr common.Address, input []byte, backend ContractBackend, unpack func([]byte) (T, error)) (arg T, err error) {
```

Comment:
dox me pls

Reply:
I don't get it :)

Reply:
These methods should have some documentation